kDrive/UI/Controller/Files/Rights and Share/RightsSelectionViewController.swift

```diff
@@ -47,7 +47,7 @@ struct Right {
     var documentDescription: String
 
     static let shareLinkRights = [
-        Right(key: ShareLinkPermission.restricted.rawValue,
+        Right(key: ShareLinkPermission.password.rawValue,
```

Comment:
`restricted` seems to be an old value used by the API and no longer used. (error when I use it)
I removed it.
Is it the case everywhere, or just this endpoint ?